tools/build/pack_cordova_sample.py

```diff
@@ -402,6 +402,18 @@ def packMobileSpec(app_name=None):
     os.chdir(orig_dir)
     return True
 
+def copySpacedodgeSource(app_name, target_path):
+    if checkContains(app_name, "SPACEDODGE"):
+        spacedodge_source_path = os.path.join(BUILD_PARAMETERS.pkgpacktools, "space-dodge-game", "base")
```

Comment:
Does the source code clone from github directly? Or we have to maintain it locally?